django_netjsonconfig/tests/test_controller.py

```diff
@@ -468,12 +468,12 @@ def test_device_report_status_no_config(self):
 class TestConsistentRegistrationDisabled(TestCase):
     @classmethod
     def setUpClass(cls):
-        super(TestConsistentRegistrationDisabled, cls).setUpClass()
```

Comment:
I actually doubt on this cls's is this correct ?

Reply:
Oh, I just checked this as well. 
I am not sure that it'll work because it's a classmethod (notice the decorator).
But I'll have to test it, did you test? :smile:

Reply:
Testing it :smile:

Reply:
Hmm! I think that wont work rolling it back :smile_cat:

Reply:
Rolled it back :smile: 

https://stackoverflow.com/questions/1817183/using-super-with-a-class-method